packages/babel-preset-env/data/plugin-features.js

```diff
@@ -90,6 +90,9 @@ const es = {
   "proposal-async-generator-functions": "Asynchronous Iterators",
   "proposal-object-rest-spread": "object rest/spread properties",
   "proposal-unicode-property-regex": "RegExp Unicode Property Escapes",
+
+  "proposal-json-strings": "", // Awaiting mapping in compat-table
```

Comment:
Just added note about this in compat-table issue 
https://github.com/kangax/compat-table/issues/1034#issuecomment-411505478